cmis/src/main/java/nl/nn/adapterframework/extensions/cmis/CmisSender.java

```diff
@@ -436,15 +430,13 @@ else if (getProperties) {
 				if(getDocumentContent) {
 					ContentStream contentStream = document.getContentStream();
 					InputStream inputStream = contentStream.getStream();
-					if (StringUtils.isNotEmpty(getFileInputStreamSessionKey())) {
-						session.put(getFileInputStreamSessionKey(), inputStream);
-					} else {
-						byte[] bytes = Misc.streamToBytes(inputStream);
 
-						if(convert2Base64)
-							session.put(getFileContentSessionKey(), Base64.encodeBase64String(bytes));
-						else
-							session.put(getFileContentSessionKey(), bytes);
+					if(convert2Base64) {
+						byte[] bytes = Misc.streamToBytes(inputStream);
+						session.put(getFileSessionKey(), Base64.encodeBase64String(bytes));
```

Comment:
Je kan hier ook een base64 inputstream in de session opslaan. Als je geen base64 doet, sla je ook een inputstream op.